.github/workflows/build-release.yml

```diff
@@ -84,3 +90,8 @@ jobs:
           path: |
             release/**/*.deb
             release/**/*.rpm
+
+      - uses: lucasmotta/pull-request-sticky-header@1.0.0
```

Comment:
cool :)

Reply:
I want a footer alternative of this one 😅

Reply:
WIP lol https://github.com/kyranjamie/pull-request-sticky-footer